sdk/storage/Azure.ResourceManager.Storage/CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Release History
 
-## 1.6.0-beta.1 (Unreleased)
+## 1.6.0 (Unreleased)
 
 ### Features Added
 
+- Upgraded api-version tag from 'package-2025-01' to 'package-2025-06'. Tag details available at https://github.com/Azure/azure-rest-api-specs/blob/1e7684349abdacee94cbf89200f319cd49e323f2/specification/storage/resource-manager/readme.md.
+    - Supported new property GeoPriorityReplicationStatus.IsBlobEnabled on Storage accounts.
+    - Supported new ObjectReplicationPolicy Properties: PriorityReplication.Enabled.
+
 ### Breaking Changes
```

Comment:
Remove all these empty entries

Reply:
Seems no unnecessary empty entries.
Could you please point out which line need be removed?